lib/svgo/js2svg.js

```diff
@@ -64,15 +66,26 @@ function JS2SVG(config) {
     this.config.indent = '    ';
   }
 
+  if (typeof this.config.eol === 'string' && this.config.eol) {
```

Comment:
this.config.eol != null should be enough here

Reply:
Addressed in 21bb65f.